Jormungand.DiscordReader-Mouth/src/main/java/entities/JorgMember.java

```diff
@@ -2,16 +2,20 @@
 
 import net.dv8tion.jda.api.entities.Member;
 import net.dv8tion.jda.api.entities.Role;
-import net.dv8tion.jda.api.entities.User;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class JorgMember {
     private JorgUser user;
-    private List<Role> roles; //Strings of JDA role names, role colours doesn't matter
+    private List<JorgRole> roles; //Strings of JDA role names, role colours doesn't matter
 
     public JorgMember(Member member){
         this.user = new JorgUser(member.getUser());
-        this.roles = member.getRoles();
+        this.roles = new ArrayList<JorgRole>();
+        for(Role role : member.getRoles()){
+            JorgRole jr = new JorgRole(role);
+            this.roles.add(jr);
+        }
     }
```

Comment:
one time `List` and `ArrayList`, please keep it unified, no big reason to use another over the other anyway